agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/models/MonitorDomain.java

```diff
@@ -52,4 +62,34 @@ public MonitorDomain setVersion(int version) {
     this.version = version;
     return this;
   }
+
+  /**
+   * Get the additionalProperties property: The abstract common base of all domains.
+   *
+   * @return the additionalProperties value.
+   */
+  @JsonAnyGetter
+  public Map<String, Object> getAdditionalProperties() {
+    return this.additionalProperties;
+  }
+
+  /**
+   * Set the additionalProperties property: The abstract common base of all domains.
+   *
+   * @param additionalProperties the additionalProperties value to set.
+   * @return the MonitorDomain object itself.
+   *     <p>//TODO (kryalama) add data sanitization
+   */
+  public MonitorDomain setAdditionalProperties(Map<String, Object> additionalProperties) {
```

Comment:
@trask do we need to sanitize properties here for consistency? I dont see any limits specified in swagger file though

Reply:
we don't use these (and I don't know what they're for) so I think fine to not sanitize